dev-tools/mage/pkgtypes.go

```diff
@@ -529,7 +529,7 @@ func (s PackageSpec) rootDir() string {
 
 	// NOTE: This uses .BeatName instead of .Name because we wanted the internal
 	// directory to not include "-oss".
-	return s.MustExpand("{{.BeatName}}{{if .FIPS}}-fips{{end}}-{{.Version}}{{if .Snapshot}}-SNAPSHOT{{end}}{{if .OS}}-{{.OS}}{{end}}{{if .Arch}}-{{.Arch}}{{end}}")
+	return s.MustExpand("{{.BeatName}}-{{.Version}}{{if .Snapshot}}-SNAPSHOT{{end}}{{if .OS}}-{{.OS}}{{end}}{{if .Arch}}-{{.Arch}}{{end}}")
```

Comment:
perhaps here is where is misses the `-fips` for the folder name

Reply:
The removal of the `{{if .FIPS}}-fips{{end}}` is intentional as this should be coming from the spec, not the Environment variable/global flag).

The reason for the difference is the pre-existing usage of `{{.BeatName}}` instead of `{{.Name}}` which, by looking at the comment, is intentional.